src/test/regress/Makefile

```diff
@@ -120,6 +122,9 @@ check-failure-base: all
 	$(pg_regress_multi_check) --load-extension=citus --mitmproxy \
 	-- $(MULTI_REGRESS_OPTS) --schedule=$(citus_abs_srcdir)/failure_base_schedule $(EXTRA_TESTS)
 
+check-upgrade:
+	$(pg_upgrade_check) --old-bindir=$(old-bindir) --new-bindir=$(new-bindir) --pgxsdir=$(pgxsdir)
```

Comment:
I think if we are going to have a `base_upgrade_schedule` and use `EXTRA_TESTS` argument we will need to add it here. We can give schedules from here as well instead of putting them to the script.

Reply:
I have been thinking a bit more on `EXTRA_TESTS` and it will be non-trivial. Normally we run 1 schedule which would in this case be a base schedule, or even an empty on given the setup is done outside of the regression test.

However, we need 2 extra tests
- one that runs before the upgrade
- one that runs after the upgrade

we either need to supply both, or enforce strict naming scheme. Both of the solutions I don't like at the moment. So I guess we can punt that till we have too many upgrade tests and need a quicker way during development to run a selective upgrade test.